code/modules/reagents/dispenser/cartridge_presets.dm

```diff
@@ -62,6 +60,14 @@
 	lime		spawn_reagent = "limejuice"
 	watermelon	spawn_reagent = "watermelonjuice"
 
+	// Bar, coffee
+	coffee		spawn_reagent = "coffee"
+	cafe_latte	spawn_reagent = "cafe_latte"
+	soy_latte	spawn_reagent = "soy_latte"
+	hot_coco	spawn_reagent = "hot_coco"
+	milk		spawn_reagent = "milk"

```

Comment:
You've only done a crate of all of them; scroll down to PACK amd SEC_PACK and add the  there too, so they can be ordered separately.